src/linear_algebra/basic.lean

```diff
@@ -758,14 +758,15 @@ preserved under addition and scalar multiplication, then `p` holds for all eleme
   (H2 : ∀ (a:R) x, p x → p (a • x)) : p x :=
 (@span_le _ _ _ _ _ _ ⟨p, H0, H1, H2⟩).2 Hs h
 
-lemma span_eq_add_submonoid.closure {M : Type*} [add_comm_monoid M] (S : set M) :
+lemma span_eq_add_submonoid.closure (S : set M) :
   (span ℕ S).to_add_submonoid = add_submonoid.closure S :=
```

Comment:
It turns out we already have this proof as [`span_nat_eq_add_group_closure`](https://leanprover-community.github.io/mathlib_docs/algebra/algebra/basic.html#span_nat_eq_add_group_closure) - could you delete that duplicate?